tools/install_python_dependencies.sh

```diff
@@ -14,21 +14,19 @@ if ! command -v "pyenv" > /dev/null 2>&1; then
   echo "pyenv install ..."
   curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
 
-  echo -e "\n. ~/.pyenvrc" >> $RC_FILE
   cat <<EOF > "${HOME}/.pyenvrc"
 if [ -z "\$PYENV_ROOT" ]; then
   export PATH=\$HOME/.pyenv/bin:\$HOME/.pyenv/shims:\$PATH
   export PYENV_ROOT="\$HOME/.pyenv"
+  eval "\$(pyenv init --path)"
   eval "\$(pyenv init -)"
   eval "\$(pyenv virtualenv-init -)"
 fi
 EOF
+  echo -e "\nsource ~/.pyenvrc" >> $RC_FILE
```

Comment:
mac setup installs pyenv from `brew`, so .pyenvrc will never be created (nor this .bashrc line) - same on any other machine when pyenv is already installed

Reply:
Yeah that is fine as far as I can tell.  This part doesn't run if pyenv was installed by other means and everything still works (since the alternative installation sets up the same things).